force_consensus.py

```diff
@@ -178,10 +177,10 @@ async def _make_transaction(self, project, epochId):
         self._nonce += 1
         return tx_hash
 
-    @aiorwlock_aqcuire_release
+    @semaphore_then_aiorwlock_aqcuire_release
     async def _reset_nonce(self):
         self._logger.info('Resetting nonce')
-        # sleep for 15 seconds to avoid nonce collision
+        # sleep for 30 seconds to avoid nonce collision
         time.sleep(30)
```

Comment:
we should be using asyncio sleep

Reply:
No, this is intentional. I need to whole script to sleep and wait not just the current task in event loop.